### ISSUE TITLE
Overall Status of selected endpoints for status page 

### ISSUE THREAD

**Issue by GalacticAust** [louislam/uptime-kuma#693]
Feature that would be great if you can select monitors and have a overall uptime displayed for the past 30 Days 

instead of each monitor listed

**Comment by deefdragon** [comment]
I don't think this is a Duplicate of #312? It sounds like it is asking for the ability to select a few monitors in the UI and see the sum up-time. 

I am still going to cross reference #468 as this is about monitor groupings in general, and the logic used in implementing some of those would likely also be used here.
